### ISSUE TITLE
Ionic2 like readymade components for Fuse

### ISSUE THREAD

**Issue by ansarizafar** [Bartekus/FuseR#2]
Thanks for starting this amazing project. What Fuse is lacking is a set of ready made Ionic2 like components. It would be great If Fuse community can join forces to develop these components.

**Comment by bartekus** [comment]
Thats one of the goals I'm after here @ansarizafar as that's seems to be what people generally ask for.

**Comment by ansarizafar** [comment]
@Bartekus Great to hear that. I am currently working on an ionic2 app but want to switch to Fuse. Ready made components make life really easy but performance of webview based apps are not good. I would recommend a separate repository, so that components can be developed and contributed independently. Any idea when will Fuse release their package manager? I would also highly recommend GraphQL or Restfull RPCs (my own solution) for backend APIs.

**Comment by bartekus** [comment]
@ansarizafar sorry for such a long absence but I have finally been able to resume work on this so as per suggestion, I've implemented full-on GraphQL Server on top of PostgreSQL using PostGraphQL.
I've decided to start from the scratch and also implement front-end this way targeting mobile using native platform thanks to FuseTools, and getting best performance/server side rendering for the front end by utilizing react-redux-graphql and a lot of other technologies, including webpack2 and babel. For the front end I decided to go with Grommet so I'll be making FuseTools versions of the assets which sticking to their pretty rad design principles.
For now I'll close this but feel free to continue conversation as I'm quite open to any comments or ideas that you might have. Cheers!
